testdata/pkg/apis/apps/v1/types.go

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
If you wanted, instead of copying Kubernetes API types, you could create a smaller set of test APIs here (no need to flesh out full spec & status structs, for example).

Reply:
Do you mean custom APIs instead of k8s? Because for now, I just copied only a few types (not all) inside `apps` and `rbac` group for testing.